.azuredevops/pull_request_templates/branches/production.md

```diff
@@ -0,0 +1,9 @@
+## Before deploying to production, please ensure:
```

Comment:
Does this file actually do anything? We never do any PRs in azure devops, and this file is in `.azuredevops`. I don't think this file will ever get used.

Reply:
Will be moving it to the .github directory